orte/mca/errmgr/base/help-errmgr-base.txt

```diff
@@ -61,9 +62,10 @@ route found between them. Please check network connectivity
 (including firewalls and network routing requirements).
 #
 [node-died]
-ORTE has lost communication with its daemon located on node:
+ORTE has lost communication with a remote daemon.
```

Comment:
Is it useful (or even possible?) to show the error message?  I.e., can we see any kind of error code indicating *why* the remote end disconnected?